tools/pybind11NewTools.cmake

```diff
@@ -32,12 +32,22 @@ if(NOT Python_FOUND AND NOT Python3_FOUND)
     set(Python_ROOT_DIR "$ENV{pythonLocation}")
   endif()
 
-  find_package(Python 3.6 REQUIRED COMPONENTS Interpreter Development ${_pybind11_quiet})
+  # Development.Module support (required for manylinux) started in 3.18
+  if(CMAKE_VERSION VERSION_LESS 3.18)
+    set(_pybind11_dev_component Development)
+  else()
+    set(_pybind11_dev_component Development.Module OPTIONAL_COMPONENTS Development.Embed)
+  endif()
+
+  find_package(Python 3.6 REQUIRED COMPONENTS Interpreter ${_pybind11_dev_component}
+                                              ${_pybind11_quiet})
```

Comment:
Wouldn't this be better with `find_package(Python3)`?

Reply:
No, if you set a minimum (which we do), they are identical, and I prefer not sticking "3" everywhere, as Python _is_ Python 3 to me.

The only reason there's a FindPython2 and FindPython3 is so you could find them both at the same time, which turned out to be not very useful - most users would also want to find multiple version of Python 3 at the same time too if they were doing this.